iptvtools/iptv_filter.py

```diff
@@ -53,6 +53,8 @@ def parse_args():
                         help=helps.UDPXY)
     parser.add_argument('-v', '--version', action='version',
                         version=__version__)
+    parser.add_argument('-n', '--no-connectivity-check', default=defaults.NO_CONNECTIVITY_CHECK,
```

Comment:
I would prefer to use `--skip-connectivity-check` with action `store_true`, just as `--replace-group-by-source`. BTW, Order the argument alphabetically is appreciated!